docs/docker.md

```diff
@@ -8,13 +8,134 @@ Start by downloading and installing Docker CE for your platform:
+
+!!! Note
+    The following section assumes that docker and docker-compose is installed and available to the logged in user.
+
+!!! Note
+    All below comands use relative directories and will have to be executed from the directory containing the `docker-compose.yml` file.
+
+### Docker quick start
+
+Create a new directory and place the [docker-compose file](https://github.com/freqtrade/freqtrade/blob/develop/docker-compose.yml) in this directory.
+
+``` bash
+mkdir freqtrade
+cd freqtrade/
+# Download the docker-compose file from the repository
+curl https://raw.githubusercontent.com/freqtrade/freqtrade/develop/docker-compose.yml -o docker-compose.yml
+
+# Pull the freqtrade image
+docker-compose pull
+
+# Create user directory structure
+docker-compose run --rm freqtrade create-userdir --userdir user_data
+
+# Create configuration - Requires answering interactive questions
+docker-compose run --rm freqtrade new-config --config user_data/config.json
+```
+
+The above snippet will create a directory called "freqtrade" - download the latest compose file and pull the freqtrade image.
+The last 2 steps will create the user-directory, as well as a default configuration based on your selections.
+
+#### Adding your strategy
+
+The configuration is now available as `user_data/config.json`.
+You should now copy your strategy to `user_data/strategies/` - and add the Strategy class name to the `docker-compose.yml` file, replacing `SampleStrategy`.
+
+Once this is done, you're ready to launch the bot in trading mode.
+
+``` bash
+docker-compose up -d
+```
+
+#### Docker-compose logs
+
+Logs will be written to `user_data/freqtrade.log`. 
+Alternatively, you can check the latest logs using `docker-compose logs -f`.
+
+#### Database
+
+The database will be in the user_data directory as well, and will be called `user_data/tradesv3.sqlite`.
+
+#### Updating freqtrade with docker-compose
+
+To update freqtrade when using docker-compose is as simple as running the following 2 commands:
+
+``` bash
+# Download the latest image
+docker-compose pull
+# Restart the image
+docker-compose up -d
+```
+
+This will first pull the latest image, and will then restart the container with the just pulled version.
+
+!!! Note
+    You should always check the changelog for breaking changes / manual interventions required and make sure the bot starts correctly after the update.
+
+#### Going from here
+
+Advanced users may edit the docker-compose file further to include all possible options or arguments.
+
+All possible freqtrade arguments will be available by running `docker-compose run --rm freqtrade <command> <optional arguments>`.
+
+!!! Note "`docker-compose run --rm`"
+    Inluding `--rm` will clean up the container after completion, and is highly recommended for all modes except trading mode (`freqtrade trade`).
```

Comment:
```suggestion
    Including `--rm` will clean up the container after completion, and is highly recommended for all modes except the container running the bot in the trading mode (with `freqtrade trade` command).
```

Reply:
I'm not certain about the second part of this change.

That sentence seems overly complicated - and i'm not sure it's really telling what is relevant.
it mentions the "container running the bot in trading mode" - but there is really only one container involved most of the time.

What i'd like to say:
* use `--rm` for all modes except trade to keep your environment clean.
* don't use `--rm`  but instead `-d` for trade mode - that container is the "default" and should remain there.

Reply:
okay